auacm/app/modules/competition_manager/views.py

```diff
@@ -31,6 +32,82 @@ def get_competitions():
     })
 
 
+@app.route('/api/competitions', methods=['POST'])
+@login_required
+def create_competition():
+    data = request.form
+    if current_user.admin == 0:
+        return serve_error('Only admins can create competitions', 401)
+    if not data['name'] or not data['start_time'] or not data['length'] or\
+            not data['problems']:
+        return serve_error('You must specify name, startTime,' +
```

Comment:
The `+` is unnecessary; Python will [automatically concatenate strings](https://stackoverflow.com/questions/18842779/string-concatenation-without-operator-in-python)